crossdock/server/server.py

```diff
@@ -88,14 +89,42 @@ def tchannelNotImplResponse():
     return TraceResponse(notImplementedError="python <=> tchannel not implemented")
 
 
-@tchannel.thrift.register(service.TracedService)
-def startTrace(request):
-    return trace_response_to_thriftrw(service, tchannelNotImplResponse())
+def make_tchannel(port):
+    global tchannel
+    tchannel = TChannel('python', hostport="localhost:%d" % port, trace=False)
+
+    service = get_thrift_service(service_name='python')
+
+    @tchannel.thrift.register(service.TracedService, method='startTrace')
+    @tornado.gen.coroutine
+    def start_trace(request):
+        return trace_response_to_thriftrw(service, tchannelNotImplResponse())
```

Comment:
may want to raise exception instead, the `tchannelNotImplResponse()` is not the right return type anyway for `start_trace`